index.html

```diff
@@ -64,7 +64,9 @@ <h1>Save the Date</h1>
             to let us know whether or not you can attend, so we can reserve and make seating arrangements for the event.
         </p>
         <p>
-          Add to Calendar
+          Add to Calendar <br> 
+          <a target="_blank" href="https://calendar.google.com/event?action=TEMPLATE&amp;tmeid=M2pqc3U3aHN2NzRvdW1lYWxjNjJnYzcza3EgYW5pdGE4MjU5M0Bt&amp;tmsrc=anita82593%40gmail.com"><img border="0" align="center" alt="Google Calendar Logo" height=60 src="https://46p2yg36reid1zp9ic1aj732-wpengine.netdna-ssl.com/wp-content/uploads/2014/02/gcallogo2.png"></a>
+          <a target="_blank" href="/Users/anitachan/Code/Alvin-and-Anita/wedding-website/Alvin & Anita's Wedding Reception.ics"><img border="0" align="center" alt="iCal logo" height=30 src="/Users/anitachan/Code/Alvin-and-Anita/wedding-website/images/iCloud Calendar icon.png"></a>
```

Comment:
Both the hyperlink and the img src refer to a path in your computer's local file system. The ics file is in this commit, so you can use a relative path to that. You'll have to download and commit the icloud calendar icon image as well, and use a relative path to it in this img src.